test/router_test.exs

```diff
@@ -102,6 +102,33 @@ defmodule BeeBee.RouterTest do
     end
   end
 
+  describe "PUT /_update" do
+    test "updates a given short tag" do
+      :post
+      |> conn("/_add", %{"url" => "https://github.com", "short_tag" => "grape"})
+      |> Router.call([])
+
+      conn =
+        :put
+        |> conn("/_update", %{"url" => "https://reddit.com", "short_tag" => "grape"})
+        |> Router.call([])
+
+      assert conn.status == 200
+      assert %{"short_tag" => "grape"} = Jason.decode!(conn.resp_body)
+      assert %{"url" => "https://reddit.com"} = Jason.decode!(conn.resp_body)
```

Comment:
You can put these assertions together into one pattern match:
```
assert %{"short_tag" => "grape", "url" => https://reddit.com"} = Jason.decode!(conn.resp_body)
```

Even better would be to check the actual redirect that you receive for this short tag after updating it.